packages/Kokkos/src/DTK_PointCloud_def.hpp

```diff
@@ -105,8 +104,8 @@ class LocalBoundingBoxFunctor
     {
         for ( size_t d = 0; d < 3; ++d )
         {
-            dst[d] = std::numeric_limits<SC>::max();
-            dst[d + 3] = -std::numeric_limits<SC>::max();
+            dst[d] = KokkosHelpers::numericLimitsMax<SC>();
+            dst[d + 3] = -KokkosHelpers::numericLimitsMax<SC>();
```

Comment:
OK - I think we are all in  agreement then including @dalg. I will update to use the traits. I just wish all of these types of tools were in kokkos. I always forget to check and see what is in tpetra.